pkg/kubelet/metrics/metrics.go

```diff
@@ -30,30 +32,26 @@ var (
 			Help:      "Image pull latency in microseconds.",
 		},
 	)
-	// TODO(vmarmol): Implement.
-	// TODO(vmarmol): Split by source?
-	PodCount = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Subsystem: kubeletSubsystem,
-			Name:      "pod_count",
-			Help:      "Number of pods currently running.",
-		},
-	)
-	// TODO(vmarmol): Implement.
-	// TODO(vmarmol): Split by source?
-	ContainerCount = prometheus.NewGauge(
-		prometheus.GaugeOpts{
+	// TODO(vmarmol): Break down by number of containers in pod?
+	SyncPodLatency = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
 			Subsystem: kubeletSubsystem,
-			Name:      "container_count",
-			Help:      "Number of containers currently running.",
+			Name:      "sync_pod_latency_microseconds",
+			Help:      "Latency in microseconds to sync a single pod. Broken down by operation type: create, update, or sync",
 		},
+		[]string{"operation_type"},
 	)
 	// TODO(vmarmol): Containers per pod
-	// TODO(vmarmol): Latency of pod startup
 	// TODO(vmarmol): Latency of SyncPods
```

Comment:
Nothing much, it should just be a new definition and defer call. Will probably be in a followup PR :)